http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfigurationBuilder.java

```diff
@@ -195,6 +198,38 @@ protected OidcClientConfiguration internalBuild(final OidcJsonConfiguration oidc
 
         oidcClientConfiguration.setTokenSignatureAlgorithm(oidcJsonConfiguration.getTokenSignatureAlgorithm());
 
+        String tmpLogoutPath = oidcJsonConfiguration.getLogoutPath();
+        log.debugf("sysProp LOGOUT_PATH: " + (tmpLogoutPath == null ? "NULL" : tmpLogoutPath));
+        if (tmpLogoutPath != null) {
+            if (isValidPath(tmpLogoutPath)) {
+                oidcClientConfiguration.setLogoutPath(tmpLogoutPath);
+            } else {
+                throw log.invalidLogoutPath(tmpLogoutPath, LOGOUT_PATH);
+            }
+        }
+
+        String tmpLogoutCallbackPath = oidcJsonConfiguration.getLogoutCallbackPath();
+        log.debugf("sysProp LOGOUT_CALLBACK_PATH: " + (tmpLogoutCallbackPath == null ? "NULL" : tmpLogoutCallbackPath));
+        if (tmpLogoutCallbackPath != null) {
+            if (tmpLogoutCallbackPath.startsWith("http")) {
```

Comment:
I don't think this check is needed. In the spec, I see, "This URI SHOULD use the https scheme...it MAY use the http scheme...The URI MAY use an alternate scheme..."

Reply:
done